engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -383,19 +375,16 @@ bool BaseRenderOpenGL3D::setup3D(Camera3D *camera, bool force) {
 			glGetFloatv(GL_MODELVIEW_MATRIX, _lastViewMatrix.getData());
 		}
 
-		bool fogEnabled;
-		uint32 fogColor;
-		float fogStart;
-		float fogEnd;
+		FogParameters fogParameters;
+		_gameRef->getFogParams(fogParameters);
```

Comment:
Since FogParams is now a struct, getFogParams could be rewritten to simply return the value, instead of the by-reference writing (which made sense when we wanted to return multiple values into multiple variables.